src/components/NewMessageModal.vue

```diff
@@ -195,6 +206,10 @@ export default {
 		this.updateCookedComposerData()
 	},
 	methods: {
+		onMaximize() {
+			this.largerModal = !this.largerModal
+			localStorage.setItem('modalSize', this.largerModal ? 'large' : 'normal')
```

Comment:
The problem with local storage is that it's gonna be only remembered on the same browser. I think we better user user Preferences here

Reply:
ok, good point